contrib/start_lint

```diff
@@ -6,6 +6,8 @@ echo "Preparing application..."
 bundle exec rake dev:prepare RAILS_ENV=test
 echo "Running linter..."
 bundle exec rake dev:lint RAILS_ENV=test
+tmp_exit=$?
```

Comment:
please no Perl variables! :sob: Use the equivalent `$CHILD_STATUS` from the [English library](https://ruby-doc.org/stdlib-2.5.1/libdoc/English/rdoc/English.html). You need to include the library.

Rubocop should also complain.


BTW, it seems that the Ruby core team finally plans to remove Perl variables in Ruby 3 :tada:

Reply:
@Ana06 this is bash, not ruby :smile:

Reply:
ouch... :see_no_evil: